src/models/site/usePutSiteSettings.js

```diff
@@ -3,10 +3,10 @@ import queryKeys from '../../constants/queryKeys';
 
 export default function usePutSiteSettings() {
```

Comment:
Any reason you chose tokeep this rather than using usePutSiteSetting with a property of ''?